test/sanitize_json.d

```diff
@@ -0,0 +1,231 @@
+import std.typecons : Flag, Yes, No;
+import std.array : Appender, appender;
```

Comment:
Only `appender` is used. It's generally a best practice to do local imports - that way it's easy not to forget to update them when code gets modified / refactored.